src/Outsystems.SetupTools/Functions/Get-OSPerfTuningStatus.ps1

```diff
@@ -0,0 +1,309 @@
+
+                    $ShouldBeSkipped = $(-not $HasSmashableMaxRequestLength) -or $(-not $HasSmashableExecutionTimeout)
+
+                    $CompTextMaxRequestLength = $(GetIsOrIsntText $HasSmashableMaxRequestLength)
+
+                    $Messages += @("Current upload limit value ('$CurrentMaxRequestLengthInMB') $CompTextMaxRequestLength the default ('$DefaultMaxRequestLengthInMB').")
+
+                    $CompTextExecutionTimeout = $(GetIsOrIsntText $HasSmashableExecutionTimeout)
+
+                    $Messages += @("Current execution timeout value ('$CurrentExecutionTimeout') $CompTextExecutionTimeout the default ('$DefaultExecutionTimeout').")
+                }
+                catch
+                {
+                    $Messages = @("Unable to determine current values. Skipping.")
+                }
+
+                return $(CreateResult -ShouldBeSkipped $ShouldBeSkipped -Messages $Messages)
+            }
+
+        $PerfTuning.Sections.IISUploadSizeLimitsConfig = CreatePerfTuningStatus -Title "IIS upload size limits configuration" `
+            -ScriptBlock `
+            {
+                $ShouldBeSkipped = $True
+                $Messages = @()
+
+                try
+                {
+                    $Filter = "system.webServer/security/requestFiltering/requestLimits"
+                    $Name = "maxAllowedContentLength"
+
+                    $MaxAllowedContentLength = $(GetWebConfigurationProperty -PSPath "MACHINE/WEBROOT/APPHOST" -Filter "$Filter" -Name "$Name")
+                    # MaxAllowedContentLength default value is 30000000
+                    #  https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/requestfiltering/requestlimits/#configuration
+                    $DefaultMaxAllowedContentLength = 30000000
+
+                    $ShouldBeSkipped = $MaxAllowedContentLength -ne $DefaultMaxAllowedContentLength
+
+                    $CompTextMaxAllowedContentLength = $(GetIsOrIsntText (-not $ShouldBeSkipped))
+
+                    $Messages = @("Current max allowed content length value ('$($MaxAllowedContentLength.Value) Bytes') $CompTextMaxAllowedContentLength the default ('$DefaultMaxAllowedContentLength Bytes').")
+                }
+                catch
+                {
+                    $Messages = @("Unable to determine current value. Skipping.")
+                }
+
+                return $(CreateResult -ShouldBeSkipped $ShouldBeSkipped -Messages $Messages)
+            }
+
+        $PerfTuning.Sections.IISConnectionsConfig = CreatePerfTuningStatus -Title "IIS for unlimited connections configuration" `
+            -ScriptBlock `
+            {
+                $ShouldBeSkipped = $True
+                $Messages = @()
+
+                try
+                {
+                    $IISConfigs = $(GetWebConfigurationProperty -PSPath "IIS:\" -Filter "system.applicationHost/sites/site[@name='Default Web Site']" -Name "Limits")
```

Comment:
(not so sure about this one) We should move this to a constant in src/Outsystems.SetupTools/Lib/Constants.ps1